lib/src/base-styles/_nx-tables.scss

```diff
@@ -201,3 +201,9 @@
     color: #045dd1;
   }
 }
+
+.nx-cell--input--filter {
+  input::-webkit-calendar-picker-indicator {
+    opacity: 100;
```

Comment:
As mentioned in WW, we aren't going to attempt to style the native `<datalist>` UI, so this should be removed.

Reply:
yes, removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/0bbcbec6c0e1241e3e197f8d22874f16aaf4fad5